src/style/_layout.scss

```diff
@@ -251,9 +251,11 @@
     {
         font-size: 16px;
 
-        display: flex;
```

Comment:
Any reason why `display: flex` was removed? AFAIK, the `flex` property doesn't implicitly trigger `display: flex` on the element.

Reply:
The flex context is enabled via its direct parent and I didn't see a reason for the context to be passed further down to its children. So I removed it as unnecessary. It does no harm to keep it, if you foresee a future need for it.